{{cookiecutter.project_name}}/docker/docker-compose.prod.yml

```diff
@@ -27,7 +27,11 @@ services:
 
   web:
     <<: &web
-      image: "registry.gitlab.com/{{ cookiecutter.organization }}/{{ cookiecutter.project_name }}:latest"
+      image:
+        "registry.gitlab.com/\
```

Comment:
Is this a linting recommendation? I don't like it, really.
I guess I have to go through the config values to find out the perfect fit.